docs/src/tutorials/GenerativeDesigns.jl

```diff
@@ -249,7 +257,7 @@ designs = efficient_designs(
     6,
     evidence;
     solver,
-    mdp_options = (; max_parallel = 2, costs_tradeoff = [0, 1.0]),
+    mdp_options = (; max_parallel = 2, costs_tradeoff = (0, 1.0)),
```

Comment:
costs_tradeoff has to be tuple? I somehow remember I just use it as a list before

Reply:
Yeah, we would like to stick to tuples after https://github.com/Merck/CEED.jl/pull/3.